files/vsc.csh

```diff
@@ -2,22 +2,19 @@ eval `/usr/bin/vsc_env csh`
 
 setenv LMOD_SYSTEM_NAME "${VSC_INSTITUTE_CLUSTER}-${VSC_ARCH_LOCAL}${VSC_ARCH_SUFFIX}"
 
-setenv CLUSTER_MODULEPATH "/etc/modulefiles/vsc"
+set CLUSTER_MODULEPATH = "/etc/modulefiles/vsc"
```

Comment:
what does this change do?

Reply:
this changes CLUSTER_MODULEPATH from an environment variable to a normal variable.
we don't need CLUSTER_MODULEPATH for anything else, so it makes more sense to use a normal var.

besides, we unset it at the end of the script with `unset CLUSTER_MODULEPATH`, which is broken for envvars: you need `unsetenv` for that.